api/routes/api.php

```diff
@@ -18,3 +20,79 @@
     Route::post('/tickets', 'createTicket');
 });
 
+/**
+ * Note: These routes are for testing/example and should be
+ * removed before merging
+ */
+Route::prefix('notify')->group(function () {
+    $templates = config('notify.templates');
+
+    Route::get('email', function () use ($templates) {
+        try {
+            $response = Notify::sendEmail(
+                'domain@test.com', // Replace with your email
+                $templates['email'],
+                ['name' => 'Your Name']
+            );
+
+            return $response->json();
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+        }
+    });
+
+    Route::get('email/attachment', function () use ($templates) {
+        try {
+            $file = Storage::get('example.txt'); // Add a file by this name to `api/storage/app`
+            $response = Notify::sendEmail(
+                'domain@test.com', // Replace with your email
+                $templates['email'],
+                ['name' => 'Your Name'],
+                null,
+                null,
+                [
+                    'file' => base64_encode($file),
+                    'filename' => "attachment.txt"
+                ]
+            );
+
+            return $response->json();
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+        }
+    });
+
+    Route::get('sms', function () use ($templates) {
+        try {
+            $response = Notify::sendSms(
+                '5555555555', // Replace with your phone number
+                $templates['sms'],
+                ['name' => 'Your Name']
+            );
+
+            return $response->json();
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+        }
+    });
+
+    Route::prefix('bulk')->group(function () use($templates) {
+        Route::get('email', function () use ($templates) {
+            try {
+                $response = Notify::sendBulk(
+                    'Test',
+                    [
+                        ['email address', 'name'],
```

Comment:
I found the bulk stuff a bit confusing, not sure of a better solution though. What about separating the concerns for email and sms for bulk?

Reply:
Yeah, I agree on this. I was struggling to figure out how to do this. Maybe an associative array?

```php
  Notify::sendBulkEmail(
    'Test',
    [
       [ 
          'email' => 'email@domain',
          'personalisation' => ['name' => 'Test']
       ]
    ]
  );
  
    Notify::sendBulkSms(
    'Test',
    [
       [ 
          'phone_number' => '5555555555',
          'personalisation' => ['name' => 'Test']
       ]
    ]
  );
```

Reply:
I think that having two methods is definitely an improvement.

Reply:
okii okii, I have updated to split the bulk sms and email as well as changing the shape of the `$rows` arg to make it a little more readable....hopefully 🤞 

[update bulk notify API to split sms and email](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5350/commits/850584e945c34cf4387df9a7acfa5b74c26dbc33)

See the API in the `NotifyTest`

https://github.com/GCTC-NTGC/gc-digital-talent/blob/850584e945c34cf4387df9a7acfa5b74c26dbc33/api/tests/Unit/NotifyTest.php#L66-L90

https://github.com/GCTC-NTGC/gc-digital-talent/blob/850584e945c34cf4387df9a7acfa5b74c26dbc33/api/tests/Unit/NotifyTest.php#L103-L127

Reply:
Nice! I would just suggest adding some details in the docstrings about the shape of `$rows`.

Reply:
Great idea, added in [cleanup + improve docs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5350/commits/2e8b6ca832c0d8ad2d923e837fece269b7f02663). Hopefully it is clear enough 😅